### ISSUE TITLE
Kodu 19.0 matrix

### ISSUE THREAD

**Issue by ts2art** [Tempest0580/tempest0580.github.io#5]
I updated to Kodi 19 Matrix and alot of my addins including temptv and tempest dont work! I reinstalled 18.9, just curious if you guys were aware of this. Thanks

**Comment by Tempest0580** [comment]
Yes, I'm in the process of updating both addons to be compatible with Kodi 19. Thanks

**Comment by ts2art** [comment]
Awesome.... Thank you

**Comment by lavara123** [comment]
@Tempest0580 What's the latest?
